src/containers/PageBuilder/Field/Field.helpers.js

```diff
@@ -41,6 +41,14 @@ export const exposeLinkProps = data => {
   return cleanUrl ? { children: linkText, href: cleanUrl } : {};
 };
 
+export const exposeSocialMediaProps = data => {
+  const { platform, url } = data;
+  const hasCorrectProps = typeof url === 'string' && url.length > 0;
+  const cleanUrl = hasCorrectProps ? sanitizeUrl(url) : null;
+  // TODO: Figure out how to validate platform!
```

Comment:
One simple way is to do something like this:
`const validPlatform = ['facebook', 'twitter'].includes(platform) ? platform : null;`

I.e. explicitly add all the supported platforms to the array.

Reply:
Exported a `supportedPlatforms` array from SocialMediaLink for this purpose and added a validation that uses that array here, thanks for the suggestion!